src/systems/pref.h

```diff
@@ -85,12 +85,21 @@ enum class U8Pref : u8 {
     SavestateClearBind,
     FalloutPlaneType,
     StageEditVariant,
+    JumpCount,
+    Weight,
+    PhysicsPreset,
+    MonkeyType,
+    JumpProfile,
 };
 
 void init();
+void tick();
 void save();
 void reset_all_defaults();
 
+bool did_pref_change(BoolPref bool_pref);
+bool did_pref_change(U8Pref u8_pref);
```

Comment:
Could use `changed` instead

Reply:
I kinda like the more descriptive wordy name, but I'll change it if you insist :P

Reply:
In general I like verbose names too, but it kinda stands out like a sore thumb among the other terse `pref` names... I could maybe go for the less redundant `did_change`

Reply:
I guess it'll be prefixed with `pref::` so sure I'll go with `pref::did_change`